### DISCUSSION TITLE
Live Teams - VueJS 16/03/22 visual code no corre comando "npm run serve"  minuto 48:00

### DISCUSSION THREAD

**Discussion by Vitaliquid** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#485] [Q&A] [answered]
Hola explorers,

instalando VueJs y VueCLI, siguiendo los pasos de la práctica del live del 16 de marzo a partir del minuto 48 del live ya no puedo seguír como muestra nuestro mission commander. recibo en la terminal de VSCode una serie de errores al intentar correr el comendo "npm run serve". les comparto los screens o si es mejor con algún log si alguien ha pasado por lo mismo. agradeceré el input. gracias y saludos!

![2022-03-22 (1)](https://user-images.githubusercontent.com/97406127/159514652-1b2d6376-e4d2-47f9-bcee-4490451219b0.png)
aquí el archivo App.vue de mi computadora tiene un comando sin terminar que salvo error no se aprecia en la sesión

![2022-03-22](https://user-images.githubusercontent.com/97406127/159514962-a917c1d1-c3df-4eaf-b229-85c8bc159885.png)
según yo estoy en el mismo paso que en el minuto 48 de la práctica del 16 de marzo.

![2022-03-22 (2)](https://user-images.githubusercontent.com/97406127/159515431-818fb9d3-118e-43cb-9cbd-1c2e82493366.png)
no logro correr el comando npm run serve para poder ver la página demo de vue a través de la dirección local que crea el mission commander en la práctica.

muchas gracias!

**Comment by doguedogue** [comment]
en lugar de usar la terminal powershell usa el **CMD o command prompt**

![image](https://user-images.githubusercontent.com/23409026/159518417-c685f51c-90ad-45e1-b6eb-c1a73e67cb12.png)

NOTA: Si se puede usar powershell pero tienes que hacer unas cosas adicionales.

**Reply by Vitaliquid** [reply]
Muchas gracias por la información pude cambiar a command prompt  y correr la ejecución de vue CLI pero sigue el mensaje de error.

**Comment by doguedogue** [comment]
Adicional usa antes de levantar tu proyecto (solo cuando se descarga o se hicieron cambios por otra persona, si es nuevo, se supone que ya viene con todos los modulos)

`npm install`

**Reply by Vitaliquid** [reply]
Gracias no se hicieron cambios y efectivamente tiene todos los módulos preinstalados.

**Comment by AngelCruzO** [comment, marked as answer]
Tienes que acceder a la carpeta que creo Vue del proyecto y correr el comando, sino nunca vas a poder visualizar tu proyecto de forma local.

Entonces tendrías que poner en tu terminal cd vue-launchx, una vez ahí npm run serve

**Reply by Vitaliquid** [reply]
Gracias, por despistado no vi que en el video el mission comander está en la carpeta vue-launchx y yo estába pidiendo el comando una carpeta antes por eso los mensajes de error, gracias nuevamente!

**Comment by ChristianDeM** [comment]
si igual pienso que no estas en la carpete indicada. de echo cuando instalas te da la instruccion de hacer un `cd`  de la carpete y  luego ya correr el comendo `npm run serve`

**Comment by gersondlarosa** [comment]
Yo también tuve ese error lo solucione entrando en al nivel que se encuentra tu proyecto de Vue.js por ejemplo puede que estés en una carpeta antes de tu proyecto y por eso te manda ese erro por que no encuentra que ejecutar

![prueba](https://user-images.githubusercontent.com/68626555/159580276-dbdcbdea-07ff-42e8-83d5-5f1e40f84b9c.png)
